libs/input-mapping/tests/ReaderTest.php

```diff
@@ -266,7 +239,7 @@ public function testReadTablesDefaultBackendBranchRewrite()
             }
         }
         foreach ($clientWrapper->getBasicClient()->listBuckets() as $bucket) {
-            if (preg_match('/^c\-[0-9]+\-docker\-test$/ui', $bucket['name'])) {
+            if (preg_match('/^c-[0-9]+-docker-test$/ui', $bucket['name'])) {
```

Comment:
cool, mel jsem za to ze to bez escapovani nefunguje 👍

Reply:
na to si stěžovalo phpcs kupodivu, ani bych si toho nevšíml sám